deploy/charts/rook-ceph-cluster/prometheus/localrules.yaml

```diff
@@ -284,7 +284,7 @@ groups:
           description: |
             The device health module has determined that the number of devices predicted to
             fail can not be remediated automatically, since it would take too many osd's out of
-            the cluster, impacting performance and potentially availabililty. You should add new
```

Comment:
For changes in the prometheus rules, we'll also want to open a PR against the ceph repo to update the [original rules](https://github.com/ceph/ceph/blob/master/monitoring/ceph-mixin/prometheus_alerts.yml).

Reply:
Are you planning on opening the ceph PR as well? We don't need to block this PR on that update, but just want to make sure it gets done.

Reply:
Yeah, it's open, but it's even bigger than rook by a significant amount. I had a couple of items open concurrently and kept pushing off all of them because ceph was huge. I think I'm ready to start.

Reply:
Wow that is a big change for ceph! I didn't imagine you'd go for an update on the entire repo. :)